_automating-configurations/api/provision-workflow.md

```diff
@@ -28,13 +28,42 @@ The following table lists the available path parameters.
 | :--- | :--- | :--- |
 | `workflow_id` | String | The ID of the workflow to be provisioned. Required. |
 
-#### Example request
+## Query parameters
+
+If you have included a substitution expression in the template, you may pass it as a query parameter or as a string value of a request body field. For example, if you specified a credential field in a template as `openAI_key: '${{ openai_key }}'`, then you can include the `openai_key` parameter as a query parameter or body field so it can be substituted during provisioning.
+
+```json
+POST /_plugins/_flow_framework/workflow/<workflow_id>/_provision?<parameter>=<value>
+```
+
+| Parameter | Data type | Description |
+| :--- | :--- | :--- |
+| User-provided | String | Parameters that match substitutions in the template. Optional. |
```

Comment:
```suggestion
| User-provided substitution expressions | String | Parameters that match substitutions in the template. Optional. |
```

Reply:
I was already making a similar edit in https://github.com/opensearch-project/documentation-website/pull/6547/commits/fda931191f7dc80ceede64da6767429ad5ca1384 :)

Reply:
The wording that you chose is perfect, thank you 😄 Let me add links